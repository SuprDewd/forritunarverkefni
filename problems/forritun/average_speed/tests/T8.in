00:06:10 84
00:07:59 175
00:13:12
00:17:07 22
00:19:31
00:20:17 23
00:27:43
00:31:49 98
00:33:42
00:40:12
00:41:14
00:48:24
00:50:45
00:58:02
01:01:14
01:07:55
01:10:04 83
01:12:14 53
01:18:07
01:26:27 82
01:27:01 75
01:28:04
01:33:04
01:35:48
01:40:00 63
01:42:47 42
01:49:53
01:52:06 14
01:55:29
02:03:38
02:08:16
02:15:55
02:22:20 97
02:22:31 38
02:22:49
02:24:45
02:27:29
02:32:35
02:34:06 144
02:42:21 184
02:44:09
02:44:57 77
02:48:50
02:53:05 72
02:56:54 33
02:59:17 190
03:01:39
03:05:39 134
03:05:51 123
03:07:23
03:11:28 14
03:14:46
03:22:53
03:28:03
03:29:45 108
03:37:42
03:42:02
03:46:30 55
03:51:50
03:59:29 30
04:06:55 173
04:13:54
04:14:15
04:17:59
04:24:09
04:31:57
04:37:27 92
04:44:31 125
04:45:15
04:50:40 181
04:52:10
04:58:24
05:06:32
05:13:57
05:17:42 122
05:23:13 60
05:23:48 28
05:27:48 154
05:28:25 1
05:33:31 80
05:33:56
05:42:05 153
05:42:19 78
05:42:55
05:44:32 80
05:49:13
05:51:32 74
05:52:14
05:54:08 0
05:57:33 107
06:03:09
06:09:40 77
06:17:44 36
06:25:18
06:26:05
06:33:51
06:37:29 115
06:41:42 176
06:48:42 50
06:52:46
06:56:02 199
07:02:11
07:08:22 128
07:13:39
07:21:03
07:27:47
07:36:06 114
07:40:56 52
07:46:41
07:53:43
07:59:53
08:05:36 179
08:08:24 82
08:12:41 94
08:16:09
08:18:41
08:26:44
08:32:53
08:36:10 165
08:42:48 72
08:48:22
08:50:35 9
08:53:49 154
08:55:07 0
09:02:18
09:08:20
09:08:24 193
09:09:19
09:15:49
09:23:19
09:30:27 22
09:33:02
09:39:09 151
09:39:22 179
09:43:37
09:48:32 37
09:56:48 97
10:04:17
10:06:11 159
10:14:25
10:19:47
10:27:37
10:30:26
10:37:41 119
10:44:22 91
10:46:48 13
10:54:28 82
11:00:35
11:06:48 178
11:09:26
11:10:48 7
11:10:51 190
11:15:35 106
11:19:12 182
11:23:39 52
11:29:20 15
11:35:06 143
11:37:11 19
11:43:47 22
11:49:01 156
11:50:54 79
11:54:01 139
11:59:23
12:02:01 121
12:05:34 29
12:05:46
12:13:23 26
12:18:15 91
12:19:58 128
12:25:54 80
12:31:33
12:31:50 33
12:34:55
12:38:20
12:41:45
12:46:12
12:49:54 58
12:52:18 9
12:59:50 184
13:00:51
13:01:07
13:05:15 24
13:07:58 153
13:13:51 6
13:17:19 25
13:17:22 184
13:24:48 47
13:27:06 16
13:31:23 139
13:37:46
13:39:05
13:42:42
13:42:49
13:48:23
13:54:17 100
13:58:37 136
13:59:43 193
14:00:54
14:04:48 147
14:06:55 63
14:08:52 110
14:15:32 12
14:22:52
14:29:22
14:30:05 125
14:36:04 161
14:43:55 144
14:46:58 66
14:47:17
14:51:35 6
14:56:31 26
14:59:23 63
15:06:59
15:13:03 141
15:19:09
15:23:20 85
15:29:31 65
15:32:46 65
15:39:04 66
15:44:55
15:46:06
15:49:22
15:53:46 200
15:54:10
15:58:24 76
15:59:03 153
16:03:32 3
16:05:39 139
16:12:52 16
16:20:42 88
16:21:51 37
16:28:02 43
16:35:42 42
16:40:46 59
16:42:32
16:46:30
16:46:56 52
16:51:57 66
16:55:32
17:02:12
17:04:45 92
17:11:24 18
17:14:51
17:18:08
17:20:21 186
17:21:50
17:28:19
17:29:04
17:36:23
17:43:54 55
17:52:12 109
