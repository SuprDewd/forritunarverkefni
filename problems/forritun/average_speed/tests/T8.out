00:13:12 17.76 km
00:19:31 30.06 km
00:27:43 33.19 km
00:33:42 37.84 km
00:40:12 48.46 km
00:41:14 50.14 km
00:48:24 61.85 km
00:50:45 65.69 km
00:58:02 77.58 km
01:01:14 82.81 km
01:07:55 93.73 km
01:18:07 105.43 km
01:28:04 114.88 km
01:33:04 121.13 km
01:35:48 124.55 km
01:49:53 137.69 km
01:55:29 140.03 km
02:03:38 141.93 km
02:08:16 143.01 km
02:15:55 144.80 km
02:22:49 146.78 km
02:24:45 148.01 km
02:27:29 149.74 km
02:32:35 152.97 km
02:44:09 179.25 km
02:48:50 186.69 km
03:01:39 205.53 km
03:07:23 221.78 km
03:14:46 230.92 km
03:22:53 232.82 km
03:28:03 234.02 km
03:37:42 248.73 km
03:42:02 256.53 km
03:51:50 269.46 km
04:13:54 300.32 km
04:14:15 301.33 km
04:17:59 312.10 km
04:24:09 329.88 km
04:31:57 352.37 km
04:45:15 380.59 km
04:52:10 396.40 km
04:58:24 415.20 km
05:06:32 439.74 km
05:13:57 462.11 km
05:33:56 489.31 km
05:42:55 501.56 km
05:49:13 509.90 km
05:52:14 513.85 km
06:03:09 526.18 km
06:25:18 552.70 km
06:26:05 553.17 km
06:33:51 557.83 km
06:52:46 592.01 km
07:02:11 615.13 km
07:13:39 646.91 km
07:21:03 662.70 km
07:27:47 677.06 km
07:46:41 708.97 km
07:53:43 715.07 km
07:59:53 720.41 km
08:16:09 745.00 km
08:18:41 748.97 km
08:26:44 761.58 km
08:32:53 771.22 km
08:48:22 801.28 km
09:02:18 807.77 km
09:08:20 807.77 km
09:09:19 810.72 km
09:15:49 831.62 km
09:23:19 855.75 km
09:33:02 879.64 km
09:43:37 895.11 km
10:04:17 926.97 km
10:14:25 951.86 km
10:19:47 966.08 km
10:27:37 986.84 km
10:30:26 994.31 km
11:00:35 1040.49 km
11:09:26 1056.79 km
11:59:23 1132.51 km
12:05:46 1145.86 km
12:31:33 1174.45 km
12:34:55 1176.52 km
12:38:20 1178.40 km
12:41:45 1180.28 km
12:46:12 1182.73 km
13:00:51 1191.33 km
13:01:07 1192.15 km
13:37:46 1261.81 km
13:39:05 1264.86 km
13:42:42 1273.24 km
13:42:49 1273.51 km
13:48:23 1286.40 km
14:00:54 1313.59 km
14:22:52 1347.06 km
14:29:22 1348.36 km
14:47:17 1389.70 km
15:06:59 1404.15 km
15:19:09 1424.85 km
15:44:55 1460.22 km
15:46:06 1461.52 km
15:49:22 1465.12 km
15:54:10 1471.29 km
16:42:32 1532.85 km
16:46:30 1536.75 km
16:55:32 1545.46 km
17:02:12 1552.80 km
17:14:51 1566.83 km
17:18:08 1567.82 km
17:21:50 1573.08 km
17:28:19 1593.18 km
17:29:04 1595.51 km
17:36:23 1618.19 km
